app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision_workflow/common.rb

```diff
@@ -20,7 +20,8 @@ def ar_ems
   # Required method to display the provision workflow in UI.
   # @param _message [String]
   # @return [void]
-  def dialog_name_from_automate(_message = 'get_dialog_name')
+  def dialog_name_from_automate(message = 'get_dialog_name')
+    super(message, {'platform' => 'ibm_vpc'})
```

Comment:
NOTE this was not implemented and was causing a spec which is common in other provider provision_workflow specs to fail :point_down: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/361/files#diff-ccaa497a4444856f0d0ba4146b3aa3102527657226abcfcfcfc1d57c21cd41e7R15-R19

For consistency with the rest of the provider's provision_workflow I decided to implemented this